docs/src/pages/components/drawers/PermanentDrawerLeft.tsx

```diff
@@ -15,53 +16,38 @@ import MailIcon from '@material-ui/icons/Mail';
 
 const drawerWidth = 240;
 
-const useStyles = makeStyles((theme: Theme) => ({
-  root: {
-    display: 'flex',
-  },
-  appBar: {
-    width: `calc(100% - ${drawerWidth}px)`,
-    marginLeft: drawerWidth,
-  },
-  drawer: {
-    width: drawerWidth,
-    flexShrink: 0,
-  },
-  drawerPaper: {
-    width: drawerWidth,
-    boxSizing: 'border-box',
-  },
+const DrawerHeader = styled('div')(({ theme }) => ({
   // necessary for content to be below app bar
-  toolbar: theme.mixins.toolbar,
-  content: {
-    flexGrow: 1,
-    backgroundColor: theme.palette.background.default,
-    padding: theme.spacing(3),
-  },
+  ...(theme.mixins.toolbar as CSSObject),
 }));
 
 export default function PermanentDrawerLeft() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box sx={{ display: 'flex' }}>
       <CssBaseline />
-      <AppBar position="fixed" className={classes.appBar}>
+      <AppBar
+        position="fixed"
+        sx={{ width: `calc(100% - ${drawerWidth}px)`, ml: `${drawerWidth}px` }}
+      >
         <Toolbar>
           <Typography variant="h6" noWrap component="div">
             Permanent drawer
           </Typography>
         </Toolbar>
       </AppBar>
       <Drawer
-        className={classes.drawer}
-        variant="permanent"
-        classes={{
-          paper: classes.drawerPaper,
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+            width: drawerWidth,
+            boxSizing: 'border-box',
+          },
         }}
+        variant="permanent"
         anchor="left"
       >
-        <div className={classes.toolbar} />
+        <DrawerHeader />
```

Comment:
Maybe we can also use `Toolbar` component?

Reply:
To be honest not sure why it was not using the `Toolbar`, maybe it’s lighter to use only the mixin 🤷‍♀️ I think it’s creating confusion. I’ll try to replace it.

Reply:
Actually it’s not the same, the mixin is adding `minHeight`

Reply:
In this demo when reviewing the use of `Toolbar` and `theme.mixins.toolbar` they have the same css that both include the `minHeight` https://next.material-ui.com/components/drawers/#responsive-drawer I'm confused.

Reply:
You were correct, I replaced `theme.mixins.toolbar` with `Toolbar`